### ISSUE TITLE
Incorrect angle brackets handling

### ISSUE THREAD

**Issue by mixaildudin** [kirbysayshi/vash#111]
Hello.
I've just come across the problem which stopped me for a couple of minutes to figure out the reason.
Here is the template's code similar to which I have:
```
@{
    var a = 1;
    var b = 2;
}

@if ((a + b) > 0) {
    <span>Some text</span>
}
```

This gives me an error saying that '>' is an unexpected token. However, if I remove the inner brackets it works just perfectly.
Thank you.

**Comment by kirbysayshi** [comment]
Definitely a bug, thanks for the report!

**Comment by kirbysayshi** [comment]
Should be fixed in v0.12.3
